packages/flutter_tools/test/web.shard/expression_evaluation_web_test.dart

```diff
@@ -113,6 +113,19 @@ void main() {
       await start(expressionEvaluation: true);
       await evaluateWebLibraryBooleanFromEnvironmentInLibrary(flutter);
     });
+
+    testWithoutContext('evaluated expression includes correctly mapped stack trace', () async {
+      await start(expressionEvaluation: true);
+      await breakInTopLevelFunction(flutter);
+      // Test that the call comes from some Dart getter called `current` (the
+      // location of which will be compiler-specific) and that the lines and
+      // file name of the current location is correct and reports a Dart path.
+      await evaluateStackTraceCurrent(flutter, <String>[
+        'get current',
+        'package:test/main.dart 24:5',
+        'package:test/main.dart 15:7',
```

Comment:
Are these the frames you would expect? Seems like there should be something outside of `package:test`.

Reply:
Talked offline but this is a red herring. The actual test package that Flutter uses is called `test` and is unrelated to the real `package:test`. :)